firmware/tools/bitmap_tools/pp_bitmap_parser.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import re
+from PIL import Image
+
+
+def parse_bitmaphpp():
+    ico_pattern = re.compile(r"static constexpr uint8_t bitmap_(.*)_data\[\] = {\n((?:\s+(?:.*)\n)+)};\nstatic constexpr Bitmap bitmap_.*\{\n\s+\{(.*)\},", re.MULTILINE)
+    ico_data = []
+    bitmaphpp_file = '/home/lupus/work/bitmap.hpp'
```

Comment:
This looks like a hard coded path, so won't work on other's computer. I think we can either:
- use a relative path, since we mostly know where the file would be run in
- use arguments to pass path
- leave it empty and wait user to fill the path (but need to handle the empty exception)

how do you think? thanks!

Reply:
I've changed the hardcoded part the same way as the main script is.